Header              header
string              node_name
int32[]             idata
float32[]           fdata
